### PR TITLE
Add a Device interface to the dbus service

### PR DESCRIPTION
**Pull request by dwlehman**
This adds a subtree `/com/redhat/Blivet1/Devices/` with individual devices' object paths being based on their `id`, eg: `/com/redhat/Blivet1/Devices/10`. As of now I don't see any need for various interfaces for device and format types, so all devices only implement `com.redhat.Blivet1.Device`.

I also added an `ObjectManager` interface at `/com/redhat/Blivet1` and moved the `Blivet` object to `/com/redhat/Blivet1/Blivet`. Other services use `Manager` for this, so I'm open to doing that instead if people think it would be better.

Things I already know I need to do:
- more unit tests
- class and module docstrings
- be a bit more stringent about checking return values, types, &c

### CODE REVIEW

**Comment by dwlehman** [comment]
- fixed unit test for ListDevices
- slightly expanded commit messages for c8a8eb8 and 0caf942

File: examples/dbus_client.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
+
+import dbus
+
+bus = dbus.SystemBus()
+bus.add_match_string("type='signal',sender='com.redhat.Blivet1',path_namespace='/com/redhat/Blivet1'")
```
Comment (vpodzime) [inline examples/dbus_client.py]: I don't quite understand what this is supposed to do. Maybe a comment would be nice here?

**Comment by vpodzime** [comment]
Other than the nitpick above this looks good to me. Very nice :)

**Comment by dwlehman** [comment]
Changed since last version:
- added some basic tests for `DBusObject` and `DBusDevice`
- added a few comments and docstrings (including the set-match in the client example)

**Comment by dwlehman** [comment]
Jenkins, please test this.

**Comment by dwlehman** [comment]
Jenkins, old chap, test this please.

File: blivet/dbus/device.py [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+        props = {"Name": self._device.name,
+                 "Path": self._device.path,
+                 "Type": self._device.type,
+                 "Size": dbus.UInt64(self._device.size),
+                 "ID": self._device.id,
+                 "UUID": self._device.uuid or "",
+                 "Status": self._device.status or False,
+                 "RaidLevel": self._get_raid_level(),
+                 "Parents": dbus.Array((self.get_object_path_by_id(d.id) for d in self._device.parents), signature='s'),
+                 "Children": dbus.Array((self.get_object_path_by_id(d.id) for d in self._device.children), signature='s'),
```
Comment (vpodzime) [inline blivet/dbus/device.py]: Shouldn't this have a signature ``'ao'`` and thus the call ``signature='o'``? I have always found the ``'o'`` and ``'s'`` signatures confusing even though I have known the difference.

**Comment by vpodzime** [comment]
Looks good to me otherwise.

File: blivet/dbus/device.py (hunk 2 above)
Comment (dwlehman) [inline blivet/dbus/device.py]: You're correct. Nice catch. I forgot to update the signature when I changed the value.

**Pull request merged**